README.md

```diff
@@ -13,11 +13,18 @@ Kubernetes on OpenStack with the help of Terraform and RancherKubernetesEngine (
 Using `clouds.yaml`:
 
 ```shell
-export OS_CLOUD=<section>
+export TF_VAR_openstack_cloud=<section>
```

Comment:
we should let this option and add another possibility using `TF_VAR_`

Reply:
Actually, I think we can have the best of both ways:
`export TF_VAR_openstack_cloud=$OS_CLOUD`

If you need/want to set variables via shell, then you can set this to either a string or just use the content of the `OS_CLOUD` variable (if it is defined).

But I would like to give users the possibility to not need environment variables at all, but rather have everything in  terraform.tfvars. (Which I find is a lot less error prone when switching between shells and projects).

I already changed the openrc.sh example accordingly.